### PR TITLE
tarea 2 Ana Agudelo

### PR DESCRIPTION
**Pull request by ana19051**

### CODE REVIEW

File: ana-agudelo2/tarea-2/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <title>Ana Agudelo - Tarea 2</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="index.css"/>
+    </head> 
+    
+    <body>
+        <main>
+            <section class="grid-container">
+                <div class="item1">August 2022</div>
```
Comment (emolinaab) [inline ana-agudelo2/tarea-2/index.html:13]: @ana19051 , avoid the use of divs, all these under grid-container should ul within li tags

**Review by emolinaab — CHANGES_REQUESTED** [review]

File: ana-agudelo2/tarea-2/index.html (hunk 1 above)
Comment (ana19051) [reply ana-agudelo2/tarea-2/index.html]: @emolinaab I changed the mistakes 👍

File: ana-agudelo2/tarea-2/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <title>Ana Agudelo - Tarea 2</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="index.css"/>
+    </head> 
+    
+    <body>
+        <main>
+            <section class="grid-container">
+                <li class="item1">August 2022</li>
```
Comment (emolinaab) [inline ana-agudelo2/tarea-2/index.html:13]: @ana19051 , li tags should be under ul tags https://www.w3schools.com/html/html_lists.asp

Comment (ana19051) [reply ana-agudelo2/tarea-2/index.html]: @emolinaab Ok, I solved the problem and I learned to use the ul and li. Thanks a lot.

**Review by emolinaab — APPROVED** [review]

**Pull request merged**